.github/labeler.yml

```
@@ -0,0 +1,20 @@
+tests:
+  - "*/**/*.spec.js"
```

```suggestion
  - "**/*.spec.js"
```

Couldn't that be just that?